### PR TITLE
[#22] | add statistic result

### PR DESCRIPTION
**Pull request by Abizrh**
---
title: Issue #22 | add statistic result
---

## What type of PR is this? (select all that apply)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 🚧 Breaking Change
- [ ] 🧑‍💻 Code Refactor
- [ ] 📝 Documentation Update

## Description

- create result page for showing user statistic data
- I use chart.js for statistic data

- I just update the code
- previously I use chart.js, then I changes into recharts so we only have one library of charts
- change the structure of my file.

The stats data is still dummy, I have not connect to db yet.

## Related Tickets & Documents

- Related Issue #
- Closes #

## QA Instructions, Screenshots, Recordings
update-code

### UI accessibility concerns?

So basically, if you completed your typing test then I'll change the page to the result chart

## Added/updated tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

### CODE REVIEW

**Comment by Killbyfire** [comment]
I made the dashboard using recharts, how about we just use recharts since its pretty easy and looks nice.

**Comment by Abizrh** [comment]
> I made the dashboard using recharts, how about we just use recharts since its pretty easy and looks nice.

ohh so that is ur pr for issue #58, is that included the issue #22?

**Comment by Abizrh** [comment]
hold for merge

**Comment by Abizrh** [comment]
@webdevcody ready for merge
